### DISCUSSION TITLE
Would be nice to be able to preview what the changes are between the current version and the edited version before publishing

### DISCUSSION THREAD

**Discussion by gabador-mof** [umbraco/Umbraco.Workflow.Issues#41] [Ideas]
Currently, approvers don't get to see where the changes have been made and can't make an informed decision about whether to approve or not. It would be nice to see the differences between the published and the current view in a side by side view.

**Comment by nathanwoulfe** [comment]
The licensed version does include the ability to view differences between the published and pending versions, but uses JsDiff (same as the CMS), which isn't great for non-primitive property values. I'm not sure of the practicality of showing two previews side-by-side, but am working on another feature which will require improved preview functionality, so hopefully will manage this case at the same time.
